src/db.go

```diff
@@ -0,0 +1,72 @@
+package api
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"os"
+
+	// go mysql driver
+	_ "github.com/go-sql-driver/mysql"
+)
+
+type mysqlConfig struct {
+	// MySQL user
+	User string
+
+	// MySQL password
+	Pass string
+
+	// MySQL database
+	DB string
+
+	// MySQL host
+	Host string
+
+	// MySQL port
+	Port string
+}
+
+var mysqlConf mysqlConfig
+
+// SetMysqlConf updates mysqlConf using environment variables.
+func SetMysqlConf() {
```

Comment:
Go naming convention: avoid using verb _set/get_, and `Mysql` should be `MySQL`. Suggest naming: `SetupMySQLConfig`